src/bitdrift_public/protobuf/filter/v1/filter.proto

```diff
@@ -62,12 +64,73 @@ message Filter {
         MATCHING_ONLY = 2;
       }
 
+      message SetFieldValue {
+        message ExistingField {
+          // The name of an existing field.
+          string name = 1 [(validate.rules).string = {min_len: 1}];
+        }
+
+        oneof value {
+          // The constant string value.
+          string string_value = 1;
+          // The value of an existing field.
+          ExistingField existing_field = 2;
+        }
+      }
+
       string name = 1 [(validate.rules).string = {min_len: 1}];
-      string value = 2 [(validate.rules).string = {min_len: 1}];
+      // The value to set the field to.
+      SetFieldValue value = 2;
+      // The type of the field to set.
       FieldType field_type = 3 [(validate.rules).enum = {defined_only: true not_in: 0}];
+      // Whether a transform is allowed to override an existing field. If no and the field already exists,
+      // the transform is a no-op.
+      bool allow_overriding = 4;
+    }
+
+    // Removes a specified field(s).
+    message RemoveFields {
+      message Fields {
+        // A single field.
+        message SingleField {
+          // The name of the field to remove. If the field doesn't exist, nothing happens. If the field
+          // occurs multiple times in the list of log's field - both in captured fields and matching fields -
+          // all occurrences of the field are removed.
+          string name = 1 [(validate.rules).string = {min_len: 1}];
+        }
+
+        oneof fields_type {
+          option (validate.required) = true;
+          SingleField single = 1;
+        }
+      }
+
+      // The fields to remove.
+      Fields fields = 2;
     }
   }
 
+  // Replaces high cardinality portion of a given input with `<id>` placeholder.
+  message ReduceCardinality {
```

Comment:
Should we just turn this into a "scrub" action and allow specifying the replacement text? Seems more generic and can be used for other things?